cross-windows.nix

```diff
@@ -96,6 +95,23 @@ let tool-version-map = import ./tool-map.nix;
             "$@"
         '';
     };
+    wine-test-wrapper = pkgs.pkgsBuildBuild.writeScriptBin "${compiler.targetPrefix}test-wrapper" ''
+        #!${pkgs.pkgsBuildBuild.stdenv.shell}
+        set -euo pipefail
+        DLLS=$(mktemp -d)
+        for p in ${pkgs.lib.concatStringsSep " " dllPkgs}; do
+          find "$p" -iname '*.dll' -exec ln -sf {} $DLLS \;
+          find "$p" -iname '*.dll.a' -exec ln -sf {} $DLLS \;
+        done
+        (
+        cd $DLLS
+        for l in lib*.dll; do
+          ln -s "$l" "''${l#lib}"
+        done
+        )
+        WINEPATH=$DLLS WINEDLLOVERRIDES="winemac.drv=d" WINEDEBUG=warn-all,fixme-all,-menubuilder,-mscoree,-ole,-secur32,-winediag WINEPREFIX=$TMP ${pkgs.pkgsBuildBuild.winePackages.minimal}/bin/wine64 "$@"
+      '';
```

Comment:
This should have a comment, what are we doing here? Why do we link all the dlls around?

Reply:
Comments added.